ddtrace/contrib/starlette/patch.py

```diff
@@ -163,9 +168,53 @@ def traced_handler(wrapped, instance, args, kwargs):
     if core.get_item(HTTP_REQUEST_BLOCKED):
         raise trace_utils.InterruptException("starlette")
 
+    # If we are on a version of starlette that is less than 0.34, we need to fix the span url
+    # since starlette sets the incorrect values for path/root_path
```

Comment:
Can we link to the starlette issue here. This will make this problem easier to track
```suggestion
    # If we are on a version of starlette that is less than 0.34, we need to fix the span url
    # since starlette sets the incorrect values for path/root_path
```

Reply:
The original issue is [here](https://github.com/encode/starlette/issues/1336)